src/minisignxml/internal/utils.py

```diff
@@ -126,7 +128,8 @@ def remove_preserving_whitespace(element: Element) -> None:
 
 def base64_binary_content(element: Element) -> bytes:
     return base64.b64decode(
-        element.text.replace("\n", "")
+        (element.text or "")
```

Comment:
This is probably my lack of python experience speaking, but is there a way to just skip all of these replace statements in the null case? 

Maybe python behaves in a way where this effectively becomes a no-op in the empty string case, but if not a null check instead of supplying an empty string might be a bit cleaner.

Reply:
> skip replace

At some point, split+join us more concise… it does the same thing if the goal is to remove white space.

another option is regexp 🥶